src/metatensor/models/utils/compute_loss.py

```diff
@@ -0,0 +1,206 @@
+from typing import Dict, List
+
+import torch
+from metatensor.torch import Labels, TensorBlock, TensorMap
+from metatensor.torch.atomistic import System
+
+from .loss import TensorMapDictLoss
+from .output_gradient import compute_gradient
+
+
+def compute_model_loss(
```

Comment:
Why is this not a method of the loss functions. This seems to me a bit cumbersome if you have to create a loss instance and then always run the method by passing the loss and model again.